TimePlatformer/Assets/Scripts/EnemyHorizontalMovementScript.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class EnemyHorizontalMovementScript : MonoBehaviour {
+
+    [Range(-1, 1)] public int direction;
+
+    public float walkSpeed;
+
+    public float maxRange;
+
+    private Vector2 startPosition;
+
+    private Vector2 walkingVector;
```

Comment:
Could you please add comments above each of your variable declarations explaining their purpose?